src/styles/index.scss

```diff
@@ -1,3 +1,42 @@
-body {
-  margin: 0;
+@import './reset';
+@import './variable';
+@import './blocks/header';
+@import './blocks/nav';
+@import './blocks/card';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+main {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, $card-width);
+  grid-template-rows: repeat(2, $card-hight);
+  gap: 48px;
+}
+
+@media (max-width: 488px) {
+  main {
+    grid-template-columns: repeat(1, $card-width);
+  }
+}
+
+@media (min-width: 488px) {
+  main {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  main {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  main {
+    grid-template-columns: repeat(4, $card-width);
+  }
 }
```

Comment:
Consider moving these styles to a separate file to keep your main file clean and clear